app/controllers/staff/training_sessions_controller.rb

```diff
@@ -43,7 +44,10 @@ def update
     @current_training_session.update(changed_params)
 
     if params['dropped_users'].present?
-        @current_training_session.users -= User.where(username: params['dropped_users'])
+      @current_training_session.users -= User.where(username: params[ 'dropped_users'])
+      User.where(username: params[ 'dropped_users']).each do |user|
```

Comment:
is it enough to say `if User.where(username: params[ 'dropped_users']).each do |user|` ?